airframe-jdbc/src/main/scala/wvlet/airframe/jdbc/DbConfig.scala

```diff
@@ -124,7 +124,8 @@ case class DbConfig(
 case class PostgreSQLConfig(
     // SSL configuration for using RDS
     useSSL: Boolean = true,
-    sslFactory: String = "org.postgresql.ssl.NonValidatingFactory"
+    sslFactory: String = "org.postgresql.ssl.NonValidatingFactory",
+    sslmode: String = "verify-full"
```

Comment:
"verify-full" is set by default when `ssl` is true.

https://github.com/pgjdbc/pgjdbc/blob/4911ed072681e209423fb608b4bf2daad01bb94d/pgjdbc/src/main/java/org/postgresql/jdbc/SslMode.java#L66-L71

Reply:
It's a bit confusing because the current default sslFactory is NonValidatingFactory https://github.com/pgjdbc/pgjdbc/blob/master/pgjdbc/src/main/java/org/postgresql/ssl/NonValidatingFactory.java, but the default ssl-mode is verify-full. 

What should be the default values for them?

Reply:
If I understand my issue correctly, the following issue is same. ( The issue was closed without any action. )
https://github.com/pgjdbc/pgjdbc/issues/1459

```
Exception in thread "main" com.zaxxer.hikari.pool.HikariPool$PoolInitializationException: Failed to initialize pool: The hostname ABCDEF.rds.example.com could not be verified by hostnameverifier PgjdbcHostnameVerifier.
```